hardware.md

```diff
@@ -99,33 +99,40 @@ The board has a physical jumper to switch 3.3V power supply to the target.
 
 JTAG IO is buffered through a TXS0108 Texas Instruments bidirectional level shifter.  UART signals are connected directly to the STM32.
 
-## FTDI target
+## Black Magic Debug App
 
-It is possible to run the Black Magic Probe firmware on your PC and have it bit bang using a USB attached FTDI MPSSE capable adapter. Programming speed is a little slower, but BMP development may be faster due to no firmware upload needed and the inherent debugging messages on the starting console.
+It is possible to use the Black Magic Debug App on your PC.
```

Comment:
The wording for this and the bullet points below connect back to the above hacking/hacking.md feedback

Reply:
This doesn't appear to be changed at all?

```suggestion
The Black Magic Debug project can also be compiled as a desktop program named Black Magic Debug App.
When compiled in this way it can support using the following adaptor types to talk with your hardware:

* BMP with a firmware version of at least v1.6.2
* ST-Link v2 and v3
* FTDI-MPSSE devices
* SEGGER J-Link probes
* CMSIS-DAP complaint adaptors
```

Reply:
Ah sorry, I had interpreted this as a remark. I had not understood that you wanted something to be changed.

Reply:
No worries :)